tasks/delete_old_reconcile_jobs/delete_old_reconcile_jobs.py

```diff
@@ -26,10 +26,12 @@ def task(
     db_connect_info: Dict,
 ) -> None:
     """
-    Gets all jobs older than internal_reconciliation_expiration_days days, then deletes their records in Postgres.
+    Gets all jobs older than internal_reconciliation_expiration_days days,
+    then deletes their records in Postgres.
 
     Args:
-        internal_reconciliation_expiration_days: Only reports updated before this many days ago will be deleted.
+        internal_reconciliation_expiration_days:
+        Only reports updated before this many days ago will be deleted.
```

Comment:
I would indent this under the Arg key like you did in other instances

```
internal_reconciliation_expiration_days:
        Only reports updated before this many days ago will be deleted.
```

Reply:
updated